lib/license_finder/cli/main.rb

```diff
@@ -158,11 +164,12 @@ def check_valid_project_path
         raise "Project path '#{config.project_path}' does not exist!" unless config.valid_project_path?
       end
 
-      def aggregate_paths
+      def aggregate_paths(strict_matching = false)
```

Comment:
Can we use the config to do this instead? That way we avoid having a parameter on this method just to pass it to the project finder.